packages/select/src/vaadin-select-base-mixin.js

```diff
@@ -436,6 +436,9 @@ export const SelectBaseMixin = (superClass) =>
       if (item.disabled) {
         itemElement.disabled = item.disabled;
       }
+      if (item.className) {
+        itemElement.setAttribute('class', item.className);
```

Comment:
Could be done via property to align with the above.

```suggestion
        itemElement.className = item.className;
```